src/style.css

```
@@ -1 +1,69 @@
 /* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  padding: 0;
+}
+
+:root {
+  --color: #00acdc;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 100%;
+  max-width: 40px;
+  vertical-align: middle;
+}
+
+.navList {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

instead space-between in this case left default start and make margin for item
reset default list styles
```suggestion
   list-style: none;
```